websites/M/MangaFreak/presence.ts

```diff
@@ -0,0 +1,90 @@
+const presence = new Presence({ clientId: "1007662369058594937" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	logo = "https://i.imgur.com/1yBi5L5.jpg",
+	searchingImg = "https://i.imgur.com/OIgfjTG.png",
+	readingImg = "https://i.imgur.com/53N4eY6.png";
```

Comment:
enum would be nice

Reply:
Yeah! I didn't know about that, I will change that when I can

Reply:
Resolved in https://github.com/PreMiD/Presences/pull/6650/commits/8c708eda25c97330152a136eec3852576ba8edb5